### DISCUSSION TITLE
estendere rete zigbee - possibile?

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#2257] [Hardware & Firmware]
Ciao, 
ho un punto della casa che purtroppo non mi arriva il segnale zigbee.
Soluzione 1: la più semplice ma quella che mi piace meno, spostare il raspberry con attaccato la chiavetta sonoff in un altra stanza, più centrale rispetto alla casa. questa soluzione la facevo proprio per ultima in quanto ora il raspy è posizione con il suo ssd in quadretto dedicato alla lan, tvcc, antifurto etc etc.
Soluzione 2: ripetitore, esiste?

**Comment by NdK73** [comment]
Quasi tutti i device ZB alimentati da rete sono anche ripetitori: ZB è una rete mesh :)

**Comment by indomus** [comment]
> Ciao, ho un punto della casa che purtroppo non mi arriva il segnale zigbee. Soluzione 1: la più semplice ma quella che mi piace meno, spostare il raspberry con attaccato la chiavetta sonoff in un altra stanza, più centrale rispetto alla casa. questa soluzione la facevo proprio per ultima in quanto ora il raspy è posizione con il suo ssd in quadretto dedicato alla lan, tvcc, antifurto etc etc. Soluzione 2: ripetitore, esiste?

Ti basta un qualsiasi componente zigbee alimentato, tipo questo https://indomus.it/componenti/osram-ledvance-smart-lightining-led-zigbee-lampadina-rgbw/

**Reply by Matty1184** [reply]
Ci avevo già pensato ma ho tutte le lampade collegate con switch bticino Smart quindi non saprei dove installarla! 
Sensori qualità aria/temp/umidità invece con alimentazione consigliate qualcosa in particolare?

**Reply by 4sp3c** [reply]
ciao, se non vuoi usare un dispositivo alimentato ci sono due strade
1) prendi una usb-stick (anche come quella della sonoff che usi già) e la flashi (parolona mista italo-anglosassone) come router e non come coordinatore
2) prova a cercare in rete zigbee extender... trovi su amazon e su aliexpress delle chiavi usb che fanno solo quello

spero di essere stato utile

p.s. dimenticavo... ovvio sono alimentati usb ma in caso ti basta la presa di casa usando un qualsiasi caricabatterie del telefono

**Reply by Matty1184** [reply]
soluzione percorribile anche quella... onestamente però pensandoci bene vorrei mettere un Sensori qualità aria/temp/umidità e dal momento che lo posizionerai dove ora ho il segnale, mi basterebbe trovare un dispositivo alimentato.
attualmente la LQI è 52... sufficiente nel caso?

**Reply by snakuzzo** [reply]
E' sempre sufficiente...anche se fosse 1. Considera poi che ci sono decine di variabili che possono essere legate alla qualunque.
Ad ogni modo... https://www.zigbee2mqtt.io/devices/DIYRuZ_AirSense.html

**Reply by sky7176** [reply]
Prendi un ripetitore Ikea che oltre tutto di da una uscita usb

**Comment by Ector73** [comment]
Ciao a tutti, mi aggancio a questa discussione per sottoporvi un altra questione:
se invece il segnale zigbee volessi portarlo in garage che però dista più di 20 metri dal coordinator (dongle usb SONOFF) che attualmente uso? Il  segnale putroppo non arriva in garage (ci sono di mezzo 4 piani).
Ho a disposizione in garage però una rete cablata e collegata a quella di casa...quindi potrei usare dispositivi wifi anzichè zigbee, tuttavia preferirei utilizzare la rete zigbee che è più "snella" e sopratutto ho dei dispositivi che mi avanzano e potrei usare questi anzichè prenderne di nuovi wifi. C'è una soluzione tecnica per portare la rete zigbee anche in garage usando magari (qui dico una castroneria forse!) la rete ethernet a disposizione?

**Reply by snakuzzo** [reply]
Puoi farlo. Crei un'altra rete zigbee.
Poi sta a te scegliere come...
- Prendi un coordinator con LAN integrata, ma comunque dovresti prevedere un'istanza software in grado di gestire una seconda istanza di ZHA o Zigbee2mqtt
- Prendi un coordinator che colleghi ad un device (es. RPI) che gestirebbe la parte software (ZHA/Zigbee2mqtt) che sarebbe poi collegato alla LAN.

modi di far funzionare le cose, ce ne sono.

**Reply by Ector73** [reply]
..dalla prima soluzione desumo che Home Assistant non può gestire più di un istanza Zigbee2mqtt?
in secondo luogo per RPI intendi raspberry?
terza cosa: Grazie! :-)

**Reply by snakuzzo** [reply]
eh no...puoi gestirne n. Ovviamente poi tocca vedere se usi HA OS oppure no.
E,comunque sia,su un qualsiasi host puoi avviare tutte le istanze di zigbee2mqtt (o qualsiasi altra applicazione) che ti pare. Basta avere concetti chiari su come funziona tutto quanto...e soprattutto su come funziona mqtt.
